server/pipeline/start.go

```diff
@@ -33,30 +32,13 @@ func start(ctx context.Context, store store.Store, activePipeline *model.Pipelin
 		log.Error().Err(err).Msg("Failed to cancel previous pipelines")
 	}
 
-	if err := store.WorkflowsCreate(activePipeline.Workflows); err != nil {
```

Comment:
ah nice that could solve the issue to not be able to restart declined pipelines :)

Reply:
removed it for now as it wasn't running when approving pipelines as we were missing the already existing workflows then and ran into a unique constraint error.